src/platform/plugins/shared/dashboard/common/index.ts

```diff
@@ -7,29 +7,13 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-export type { DashboardCapabilities } from './types';
+export type {
+  DashboardCapabilities,
+  DashboardLocatorParams,
+  DashboardSettings,
+  DashboardState,
+} from './types';
 
 export type { DashboardPanelMap, DashboardPanelState } from './dashboard_container/types';
 
-export {
-  type InjectExtractDeps,
-  injectReferences,
-  extractReferences,
-} from './dashboard_saved_object/persistable_state/dashboard_saved_object_references';
-
-export {
-  createInject,
-  createExtract,
-} from './dashboard_container/persistable_state/dashboard_container_references';
-
-export { prefixReferencesFromPanel } from './dashboard_container/persistable_state/dashboard_container_references';
-
-export {
-  convertPanelsArrayToPanelMap,
-  convertPanelMapToPanelsArray,
-  generateNewPanelIds,
-} from './lib/dashboard_panel_converters';
-
-export const UI_SETTINGS = {
-  ENABLE_LABS_UI: 'labs:dashboard:enable_ui',
-};
```

Comment:
These top level exports were removed to avoid bloating the bundle size. Aside from types, the imports from common have been changed to deep imports.